treeherder/log_parser/utils.py

```diff
@@ -29,14 +29,12 @@ def extract_text_log_artifacts(project, log_url, job_guid):
     return artifact_list
 
 
-def post_log_artifacts(project, job_guid, job_log):
+def post_log_artifacts(job_log):
     """Post a list of artifacts to a job."""
-    log_url = job_log.url
-    log_description = "%s %s (%s)" % (project, job_guid, log_url)
```

Comment:
The log URL is now no longer printed in the case of errors, so the user has to go manually look it up.
This may be fine now we're using New Relic custom attributes (and so relying more on New Relic than the papertrail logs), but just wanted to check.